orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -1004,7 +1004,8 @@ TEST(GradientGraphBuilderTest, TrainingSession_PipelineTransform_base) {
   auto load_gradient_graph = [](int stageIdx, PathString& input_file, PathString& output_file) {
     auto config = MakeBasicTrainingConfig();
 
-    config.use_pipeline = true;
+    TrainingSession::TrainingConfiguration::PipelineConfiguration pipe{};
```

Comment:
can we avoid using a name pipe? 
pipe_config?

Reply:
Done. But now I need your approval again. :)